.github/workflows/release-python.yml

```diff
@@ -30,37 +30,18 @@ jobs:
       CXXFLAGS: "-std=c++11 -stdlib=libc++"
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        package-name: [qcs-sdk-python, qcs-sdk-python-grpc-web]
```

Comment:
I think you still want `python-version`, right?

Reply:
Not necessary with https://github.com/rigetti/qcs-sdk-rust/pull/449#discussion_r1548734359